### DISCUSSION TITLE
Filter by count of related records

### DISCUSSION THREAD

**Discussion by vadim-shilov** [prisma/prisma#7745] [Q&A]
Are the any plans to support filtering records by the count of related records?

Example:
```ts
// Find groups where count of students > 10 
const groups = await prisma.group.findMany({
  where: {
    students: {
      _count: {
        where: {
          ...additional filtering 
        },
        gt: 10
      }
    }
  }
});
```

**Comment by ryands17** [comment]
@vadim-shilov 👋 

I suppose this request is somewhat similar to your use case: https://github.com/prisma/prisma/issues/6570

**Reply by ryands17** [reply]
If so it would be great if you could add a 👍 so that we can look at the priority.

**Reply by vadim-shilov** [reply]
Yes! It looks like) Thx

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
